.env

```diff
@@ -29,3 +29,7 @@ ADOBE_CAMPAIGN_ORG_ID=asdf
 ADOBE_CAMPAIGN_ORG_NAME=cru
 ADOBE_CAMPAIGN_TECH_ACCT=asdf
 ADOBE_CAMPAIGN_SIGNED_JWT=asdf
+
+OKTA_SERVER_URL=https://dev1-signon.okta.com
+OKTA_SERVER_PATH=https://dev1-signon.okta.com
+OKTA_SERVER_AUDIENCE=https://dev1-signon.okta.com
```

Comment:
I'm guessing you are using this as a placeholder value?

my basic understanding of the id_token at this point is that the audience should be the client_id that the token is generated for. This client_id should be unique per app/service that uses Okta for Auth.

Reply:
Yes, these are placeholder values. The code will actually support multiple audiences/client_ids which I think we will need to have the web and apps both use Okta login.

Reply:
just wanted to be sure since this is the first time I'm digging into how Okta handles auth between a mobile app and API